pulse_actions/worker.py

```diff
@@ -59,7 +60,7 @@ def handler_with_dry_run(data, message):
         try:
             pulse.listen()
         except Exception as e:
-            LOG.info("Error: %s" % e)
+            traceback.print_exc()
 
```

Comment:
I saw errors in the log like this:
    INFO:	 Error: list index out of range 

These are not at all helpful, so I think having a traceback is a better way to log this.